### ISSUE TITLE
Starting GMusicProxy fails when gmusicproxy.net is offline

### ISSUE THREAD

**Issue by kokx** [diraimondo/gmusicproxy#45]
At the moment, gmusicproxy.net is offline. Also, GMusicProxy isn't able to start, and fails with the following error:

```
Google Play Music Proxy 1.0.5 (© Mario Di Raimondo)

Traceback (most recent call last):
  File "/usr/bin/GMusicProxy", line 665, in <module>
    checkLatestVersion()
  File "/usr/bin/GMusicProxy", line 583, in checkLatestVersion
    latestVersion = opener.open(urlLatestVersion).read()
  File "/usr/lib/python2.7/urllib2.py", line 431, in open
    response = self._open(req, data)
  File "/usr/lib/python2.7/urllib2.py", line 449, in _open
    '_open', req)
  File "/usr/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 1227, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/usr/lib/python2.7/urllib2.py", line 1197, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 111] Connection refused>
```

Though correlation doesn't imply causation, I do think that the check for the latest version fails because gmusicproxy.net is offline.

Hence, can you make a fail in the version check have no impact on the ability to run GMusicProxy?

**Comment by diraimondo** [comment]
I've pushed a fix on git master minutes ago (see 55a07fc). Try this version or, as temporary fix, disable the version check. 

Sorry!

**Comment by kokx** [comment]
Thanks! Seems like gmusicproxy.net is also online again. And no problem, these things happen sometimes.
